pkg/tools/clientmap/refcount_test.go

```diff
@@ -17,14 +19,21 @@
 package clientmap_test
 
 import (
+	"strconv"
+	"sync"
 	"testing"
 
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/networkservicemesh/sdk/pkg/networkservice/common/null"
 	"github.com/networkservicemesh/sdk/pkg/tools/clientmap"
 )
 
+const (
+	parallelCount = 1000
```

Comment:
Why 1000?

Reply:
My fault it is not actually the `parallelCount` but the `retriesCount`.
`TestRefcountMap_Actions` is not the test fully validating the correctness of the `RefcountMap`. It should **eventually** fail if `RefcountMap` is invalid. So if we start increasing retries count, such **eventually** is becoming **most probably**.
1000 is good, because it is not too big to make test performing > 1 sec (10000 is too big for such case) and not too small to keep **eventually** somewhere far away from us.